### PR TITLE
Remove @fetch and nextgen engine 

### PR DESCRIPTION
**Pull request by dondonz**
Removing the `@fetch` directive and the nextgen engine, both were deprecated some time ago.

### CODE REVIEW

File: src/main/java/graphql/execution/Execution.java [hunk 1]
```diff
@@ -175,30 +172,4 @@ private CompletableFuture<ExecutionResult> executeOperation(ExecutionContext exe
 
         return result;
     }
-
-
-    private GraphQLObjectType getOperationRootType(GraphQLSchema graphQLSchema, OperationDefinition operationDefinition) {
```
Comment (dondonz) [inline src/main/java/graphql/execution/Execution.java:180]: Moved this a util, it's also used by the `ExecutableNormalizedOperationFactory`

File: src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java [hunk 2]
```diff
@@ -109,7 +109,7 @@ private ExecutableNormalizedOperation createNormalizedQueryImpl(GraphQLSchema gr
                 .normalizedVariables(normalizedVariableValues)
                 .build();
 
-        GraphQLObjectType rootType = Common.getOperationRootType(graphQLSchema, operationDefinition);
```
Comment (dondonz) [inline src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java:112]: This was the only reference to nextgen outside of the nextgen files

File: src/main/java/graphql/schema/impl/SchemaUtil.java [hunk 3]
```diff
@@ -96,4 +103,29 @@ public static void replaceTypeReferences(GraphQLSchema schema) {
         SchemaTraverser schemaTraverser = new SchemaTraverser(schemaElement -> schemaElement.getChildrenWithTypeReferences().getChildrenAsList());
         schemaTraverser.depthFirst(new GraphQLTypeResolvingVisitor(typeMap), roots);
     }
+
+    public static GraphQLObjectType getOperationRootType(GraphQLSchema graphQLSchema, OperationDefinition operationDefinition) {
```
Comment (dondonz) [inline src/main/java/graphql/schema/impl/SchemaUtil.java:107]: Add util method here. Let me know if there's a better util to chuck this method into

Comment (bbakerman) [reply src/main/java/graphql/schema/impl/SchemaUtil.java:107]: Fair enough place.

**Review by bbakerman — APPROVED** [review]

**Comment by bbakerman** [comment]
Nice one

**Pull request merged**